planner/cascades/testdata/integration_suite_out.json

```diff
@@ -269,9 +269,10 @@
       {
         "SQL": "select b from t order by b limit 3",
         "Plan": [
-          "TopN_8 3.00 root test.t.b:asc, offset:0, count:3",
-          "└─TableReader_10 10000.00 root data:TableScan_11",
-          "  └─TableScan_11 10000.00 cop[tikv] table:t, range:[-inf,+inf], keep order:false, stats:pseudo"
+          "Limit_10 3.00 root offset:0, count:3",
+          "└─TableReader_15 3.00 root data:TopN_16",
+          "  └─TopN_16 3.00 cop[tikv] test.t.b:asc, offset:0, count:3",
```

Comment:
This is not right. It should use Limit instead of TopN. I'll investigate this.

Reply:
This shoule be `TopN->TableReader->TopN->TableScan` after addressing the comment above.